### PR TITLE
More SlotMeta docs

### PR DESCRIPTION
**Pull request by garious**
#### Problem

SlotMeta descriptions lack precision

#### Summary of Changes

Take a stab at creating at better ones

### CODE REVIEW

File: src/blocktree.rs [hunk 1]
```diff
@@ -123,20 +123,25 @@ pub trait LedgerColumnFamilyRaw {
...
+    // The number of slots above the root (the genesis block). The first
+    // slot has slot_height 0.
     pub slot_height: u64,
-    // The total number of consecutive blob starting from index 0
+    // The total number of consecutive blobs starting from index 0
     // we have received for this slot.
     pub consumed: u64,
-    // The entry height of the highest blob received for this slot.
+    // The index *plus one* of the highest blob received for this slot.
+    // Useful for checking if the slot has received any blobs yet.
```
Comment (rob-solana) [inline src/blocktree.rs]: also useful for repair, scans for holes can limit themselves to received - consumed indexes...

File: src/blocktree.rs [hunk 2]
```diff
@@ -123,20 +123,25 @@ pub trait LedgerColumnFamilyRaw {
...
+    // slot has slot_height 0.
     pub slot_height: u64,
-    // The total number of consecutive blob starting from index 0
+    // The total number of consecutive blobs starting from index 0
     // we have received for this slot.
     pub consumed: u64,
-    // The entry height of the highest blob received for this slot.
+    // The index *plus one* of the highest blob received for this slot.
+    // Useful for checking if the slot has received any blobs yet, and
+    // to calculate the length of a hole: `received - consumed`.
```
Comment (CriesofCarrots) [inline src/blocktree.rs]: I don't think this calculation actually tells you the length of the hole. Just that there is at least one hole in that range. Right?

Comment (garious) [reply src/blocktree.rs]: Yep, you're right

File: src/blocktree.rs (hunk 1 above)
Comment (garious) [reply src/blocktree.rs]: fixed

File: src/blocktree.rs (hunk 2 above)
Comment (garious) [reply src/blocktree.rs]: fixed, better? @CriesofCarrots

Comment (CriesofCarrots) [reply src/blocktree.rs]: Much 👍

**Pull request merged**